packages/cli/package.json

```diff
@@ -38,7 +38,9 @@
     "clean": "rm -rf dist",
     "prepublishOnly": "yarn clean && yarn build",
     "test": "jest",
-    "test:watch": "yarn test --watch"
+    "test:watch": "yarn test --watch",
+    "lint": "yarn eslint './src/'",
+    "lint:fix": "yarn eslint --fix './src/'"
```

Comment:
The ‘lint’ scripts are not necessary. Currently we run lint from root, targeting all packages. Additionally, these scripts use “‘“ and “/“, which will cause an error on Windows. We’re trying to resolve issues with Windows lint script and errors in #395